### DISCUSSION TITLE
File not found error in particle refinement

### DISCUSSION THREAD

**Discussion by Angeevillor** [nextpyp/.github#24] [Q&A] [answered]
Hello, I did a turtorial process on the sample data (EMPIAR-10164), everything was fine before, but when I reached the Step 6（Reference-based refinement）, the following error occurred:
<img width="1109" alt="trace back" src="https://github.com/nextpyp/.github/assets/139559515/20d041da-575f-41c3-8a8e-204d241e177d">
It looks like I'm missing a "stacks.txt" file, but when I check the input there's no interface for that file. Besides, this file seems not appear to have been generated in the previous preprocessing results.So I would like to ask how to solve this problem?
Another issue is that I notice the following command line appearing on the panel while the program is running：
<img width="1115" alt="bug" src="https://github.com/nextpyp/.github/assets/139559515/41e61789-423f-4a06-a959-3af112794225">
<img width="1026" alt="bug2" src="https://github.com/nextpyp/.github/assets/139559515/2f170a3e-8a32-4271-93f4-f4620e6126f4">
I noticed that they all run as "/opt/pyp/bin/run/pyp", but I didn't generate any files under /opt during the installation. I would also like to ask how this works?

**Comment by tofushan** [comment, marked as answer]
Hi, 

For the first question, unfortunately, we don't have enough information to tell what's going on here. Could you share more about your resource settings (i.e., Threads per task, bundle size, etc.), or share the entire log file if possible? Also, are your pre-processing working fine? 

For the second question, `/opt/pyp/bin/run/pyp` is actually the path pointing to your nextPYP used inside the singularity container. This path can be accessed inside the container but not from outside.

**Reply by abartesaghi** [reply]
Since we haven't heard back from you @Angeevillor, we assume this issue was resolved, but please let us know otherwise.
